### DISCUSSION TITLE
Collecting Bales and Deliver them to a special point; Send Tractors without Machines around the Course

### DISCUSSION THREAD

**Discussion by KugelPfusch** [Courseplay/Courseplay_FS22#1543] [Ideas]
Hi guys,
I wondered if Courseplay can already automatically unload bales at an specific point. As fas as i know the tractor just stops as soon as the Bale Loader is full. I also tried to send an Tractor with an Autoload Trailer through the course with an special offset. But you can't just send a Tractor without an "machine" to drive around the course 🤔

**Comment by Tensuko** [comment]
Only with AutoDrive. If there will be another way in the future I cant tell right now.

**Reply by KugelPfusch** [reply]
Okay thanks 😅
It would be enough if the bale loader just unloads Everything as soon as he is full but I will wait.

**Comment by fudoido** [comment]
As far as I am aware from previous communications from Courseplay team, they will NOT create driving modes, so that option is likely out of the future plans.
They stated that are working closer with autodrive team and all driving courses are meant to be used with autodrive, preferring to make both mods cooperate rather than creating modes that autodrive already performs.

**Reply by KugelPfusch** [reply]
Thank you very much. That would be awesome.
